### ISSUE TITLE
sendImageAsSticker() returns: TypeError: ((intermediate value)(intermediate value)(intermediate value) || (intermediate value)(intermediate value)(intermediate value)).replace is not a function

### ISSUE THREAD

**Issue by TVARecordings** [open-wa/wa-automate-nodejs#3215]
### Are you using the latest version of the library?

- [X] I have checked and am using the latest version of the library.

### What type of session are you experiencing this issue on?

Multi-device and I have set multiDevice to true in my config OR am using the --multi-device flag

### What type of host account are you experiencing this issue on?

Business account

### Mode

EASY API/CLI

### Current Behavior

So, since the latest update; the code is returning error: TypeError: ((intermediate value)(intermediate value)(intermediate value) || (intermediate value)(intermediate value)(intermediate value)).replace is not a function

for function: sendImageAsSticker() and all other sticker functions. 

### Expected Behavior

Expected it to create the sticker :-)

### Steps To Reproduce

Latest version. 

### DEBUG INFO

```markdown
{
  "WA_VERSION": "2.3000.1013788034",
  "WA_AUTOMATE_VERSION": "4.71.11",
  "BROWSER_VERSION": "Chrome/125.0.6422.77",
  "START_TS": 1716909032204,
  "RAM_INFO": "Total: 68.62 GB | Free: 46.74 GB",
  "PPTR_VERSION": "19.11.1",
  "LATEST_VERSION": true,
  "CLI": false,
  "PATCH_HASH": "CACHED-b94ec",
  "LAUNCH_TIME_MS": 6706,
  "ACC_TYPE": "BUSINESS",
  "chats": 33,
  "messages": 79,
  "contacts": 2265,
  "isBiz": true,
  "isMd": true,
  "INSTANCE_ID": "062b0e88-cddf-4853-82a2-93e1b99276fa"
}
```

### Environment

```markdown
- OS: Windows 10
- Node: 20.11.1
- npm: 10.2.4
```

**Comment by homam49** [comment]
did you find a solution? I'm facing the same problem

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
try {
	execSync(cmd, {stdio: 'inherit'})
}  catch (error) {
    console.log(`Status Code: ${error.status} with '${error.message}'`);
 }
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN && npm run types:publish `);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.71.12](https://github.com/open-wa/wa-automate-nodejs/compare/4.71.11...4.71.12) (2024-06-06)

- Release 4.71.12 [`060d3e5`](https://github.com/open-wa/wa-automate-nodejs/commit/060d3e58b97f5f8226c18c9e0486eddd93c0de66)
- 🐛 fix custom sticker endpoint issue #3215 [`0ec9214`](https://github.com/open-wa/wa-automate-nodejs/commit/0ec9214d3687e660019d546bbdc7688ad487f485)
- updated types-only package [`6e10622`](https://github.com/open-wa/wa-automate-nodejs/commit/6e106220ae9b3ebc32ce9559d89977944725ae7b)

**Comment by smashah** [comment]
should be fixed with latest update thanks
